client/components/data/domain-management/index.jsx

```diff
@@ -61,11 +62,16 @@ class DomainManagementData extends Component {
 						user: this.props.currentUser,
 					} ) }
 				</CalypsoShoppingCartProvider>
+				<UsePresalesChat />
 			</div>
 		);
 	}
 }
 
+export function UsePresalesChat() {
+	usePresalesChat( 'wpcom' );
+}
+
```

Comment:
Is there a better way to do this?

Reply:
You can just call it directly, it's simply a hook: https://github.com/Automattic/wp-calypso/blob/32b0c450d155b58dafe38840f697bdbd42dd9a05/client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/index.tsx#L16

Reply:
I think this was due to it being used in a class component.

Reply:
Ah, I see, I missed that!